uaclient/api/u/pro/services/enable/v1.py

```diff
@@ -96,14 +96,12 @@ def _enable(
             messages=[],
         )
 
-    ent_cls = entitlements.entitlement_factory(
-        cfg=cfg, name=options.service, variant=options.variant or ""
-    )
-    entitlement = ent_cls(
-        cfg,
+    entitlement = entitlements.entitlement_factory(
```

Comment:
we lost the `called_name` argument here

Reply:
That is by design, I am using the `name` as the same value for `called_name` in the `entitlement_factory` function.
But let me know if this should be an explicit parameter